packages/backend/src/open_payments/payment/outgoing/model.ts

```diff
@@ -194,9 +191,6 @@ export class OutgoingPayment
     if (this.error) {
       data.error = this.error
     }
-    if (this.peerId) {
-      data.peerId = this.peerId
-    }
```

Comment:
We should also remove it from the `webhooks.yaml` OpenAPI spec (`outgoingPaymentEvent` component)